packages/odyssey-react/src/components/Button/Button.test.tsx

```diff
@@ -0,0 +1,39 @@
+/*!
+ * Copyright (c) 2021-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import Button from "./Button";
+
+const buttonLabel = "Button Label";
+
+describe("Button", () => {
+  it("render the button", () => {
+    const { getByText } = render(<Button onClick={jest.fn()} variant="primary">{buttonLabel}</Button>);
+    
+    expect(getByText(buttonLabel)).toBeInTheDocument();
+  });
+
+  it('should be disabled', () => {
+    const { getByRole } = render(<Button onClick={jest.fn()} variant="primary" disabled={true}>{buttonLabel}</Button>);
```

Comment:
just wanted to mention that JSX includes syntactic sugar for this type of scenario

```suggestion
    const { getByRole } = render(<Button onClick={jest.fn()} variant="primary" disabled>{buttonLabel}</Button>);
```